swagger.yaml

```diff
@@ -4929,6 +5155,39 @@ components:
       example:
         - label: "1968"
           cbor_metadata: "\\xa100a16b436f6d62696e6174696f6e8601010101010c"
+    tx_content_redeemers:
+      type: array
+      items:
+        type: object
+        properties:
+          tx_index:
+            type: integer
+            example: 0
+            description: Index of the certificate within the transaction
+          purpose:
+            type: string
+            enum: [spend, mint, cert, reward]
+            example: "spend"
+            description: Validation purpose
+          unit_mem:
+            type: string
+            example: "1700"
+            description: The budget in Memory to run a script
+          unit_steps:
+            type: string
+            example: "476468"
+            description: The budget in Cpu steps to run a script
+          fee:
+            type: string
+            example: "172033"
+            description: The fee consumed to run the script
```

Comment:
Some of these look small enough to be ints but I'm not sure.

Reply:
Internally, `fee` is defined in a way that it can easily overflow int, so let's keep it as string.